tkg/client/management_components.go

```diff
@@ -101,7 +112,29 @@ func (c *TkgClient) InstallOrUpgradeManagementComponents(kubeconfig, kubecontext
 	return err
 }
 
-func (c *TkgClient) getTKGPackageConfigValuesFile(managementPackageVersion, kubeconfig, kubecontext string, upgrade bool) (string, error) {
+// GetAddonsManagerPackageversion returns a addons manager package version
+func (c *TkgClient) GetAddonsManagerPackageversion(managementPackageVersion string) (string, error) {
+	envDefinedVersion := os.Getenv("_ADDONS_MANAGER_PACKAGE_VERSION")
+	if envDefinedVersion != "" {
+		return envDefinedVersion, nil
+	}
+	packageVersion := managementPackageVersion
+	var err error
+	if packageVersion == "" {
+		packageVersion, err = c.tkgBomClient.GetManagementPackagesVersion()
+		if err != nil {
+			return "", err
+		}
+	}
```

Comment:
Need to remove "v" from version string. The version in BOM is `v0.28.0-dev-94-g006af5a8`. But the package version doesnt have v in it. 

Refer to this commit: https://github.com/vmware-tanzu/tanzu-framework/pull/2141/commits/67db945a4c5d0e79a87d748020773f8f3b07bf97 where `	managementPackageVersion = strings.TrimLeft(managementPackageVersion, "v")` is done

Reply:
done